src/mixins/Context.vue

```diff
@@ -1,18 +1,23 @@
 <script>
-import { mapActions } from 'pinia'
-import { useContextStore } from "@/stores/context"
+import {mapActions} from 'pinia'
+import {mapState} from 'pinia'
+import {useContextStore} from '@/stores/context'
 
 export default {
   name: 'Context',
-  setup() {
-    const context = useContextStore()
-    return {
-      currentUser: context.currentUser,
-      loading: context.loading
-    }
+  computed: {
+    ...mapState(useContextStore, [
+      'applicationState',
+      'config',
+      'currentUser',
+      'eventHub',
+      'isLoading',
+      'screenReaderAlert'
+    ]),
   },
   methods: {
     ...mapActions(useContextStore, [
+      'alertScreenReader',
       'loadingComplete',
       'loadingStart'
     ])
```

Comment:
Having this "mixin" makes sense and most of our other tools follow this pattern. However, in the Vue 3 world, `mixins` are discouraged. I don't think you should refactor (yet). Maybe down the road. 😄  See https://vuejs.org/api/options-composition.html

<img width="692" alt="image" src="https://github.com/ets-berkeley-edu/hartsfield/assets/7606442/02a44d57-a7f5-4a84-9d9d-d9ebe775a6ff">